shardingsphere-infra/shardingsphere-infra-common/src/test/java/org/apache/shardingsphere/infra/database/type/DatabaseTypeRecognizerTest.java

```diff
@@ -126,7 +126,7 @@ private DataSource mockDataSource(final DatabaseType databaseType) throws SQLExc
                 url = "jdbc:postgresql://localhost:5432/test";
                 break;
             case "SQL92":
-                url = "jdbc:sqlserver://127.0.0.1;DatabaseName=test";
```

Comment:
Hi why do you swap this URL?

Reply:
Because press the new matching pattern jdbc:sqlserver Return type SQLServer instead of SQL92